pandas/core/window/numba_.py

```diff
@@ -161,3 +164,67 @@ def groupby_ewma(
         return result
 
     return groupby_ewma
+
+
+def generate_numba_table_func(
+    args: Tuple,
+    kwargs: Dict[str, Any],
+    func: Callable[..., Scalar],
+    engine_kwargs: Optional[Dict[str, bool]],
+    name: str,
+):
+    """
+    Generate a numba jitted function to apply window calculations table-wise
+
```

Comment:
where are the expectations on the user callable here? (e.g. input / output shape)

Reply:
Good point. Added